README.md

```diff
@@ -385,7 +389,8 @@ requests and response as key-value pairs.
 ###### Request
 
 ```text
-origin=remote type=request correlation=2d66e4bc-9a0d-11e5-a84c-1f39510f0d6b protocol=HTTP/1.1 sender=127.0.0.1 method=POST path=http://example.org/test headers={Accept=[application/json], Content-Type=[text/plain]} body=Hello world!
+origin=remote type=request correlation=2d66e4bc-9a0d-11e5-a84c-1f39510f0d6b protocol=HTTP/1.1 sender=127.0.0.1 method=POST uri=http://example.org/test host=example.org scheme=http port=null path=/test headers={Accept=[application/json], Content-Type=[text/plain]} body=Hello world!

```

Comment:
I'm not an expert on splunk - maybe an empty value would be cleaner?

Reply:
Looks like it https://community.splunk.com/t5/Splunk-Search/What-is-the-best-way-to-search-for-blank-null-fields-in-a-search/m-p/211083